CHANGES.md

```diff
@@ -8,6 +8,8 @@
 * Add `roundRobin` as a load balancer option.
 * Add the ability to route basted on the `dest` request header when using the
   TTwitter Thrift protocol
+* Change the default `dstPrefix` from the protocol name to `/s`
+* Change the default HTTP identifier to simply use the `Host` header
```

Comment:
sort of... it's the io.l5d.header identifier with "host" as the header.  I think I'll just make "host" the default header for the io.l5d.header identifier which makes this much easier to explain.